cmd/manager/kodata/serving_v1alpha1_knativeserving_cr.yaml

```diff
@@ -0,0 +1 @@
+serving_v1alpha1_knativeserving_cr.yaml
```

Comment:
This is a broken link. You need to link the same way as 
../../../config/crds/serving_v1alpha1_knativeserving_cr.yaml.